### PR TITLE
Peer to peer color changer

### PR DESCRIPTION
**Pull request by jaykerbs**
- Background color changes when ball impacts paddle.
- Font color changes based on the darkness of the background color.

### CODE REVIEW

File: src/components/Pong/index.js [hunk 1]
```diff
@@ -105,10 +113,17 @@ class Pong extends Component {
...
+        const rgbValues = valuesRGB();
+        const colorValues = getRGB(rgbValues);
+        const color = setColor(rgbValues);
+        console.log(color);
+        const luminance = getLuminance(colorValues);
+        const fontColor = getFontColor(luminance);
+        console.log(fontColor);
+        this.props.onChangeColor(color, fontColor);
+        this.props.webrtc.shout('bgColor', { bgColor: color });
+        this.props.webrtc.shout('fontColor', { fontColor: fontColor });
```
Comment (lazorfuzz) [inline src/components/Pong/index.js]: Don't send fontColor when the recipient can already calculate the luminance from the bgColor they received

**Review by lazorfuzz — CHANGES_REQUESTED** [review]
Pls make sure code works when merged with current codebase before attempting PR

File: src/components/Pong/index.js (hunk 1 above)
Comment (jaykerbs) [reply src/components/Pong/index.js]: Fixed it! @lazorfuzz

**Review by Cristian006 — APPROVED** [review]
The naming of functions can be improved but it works 👍

**Pull request merged**